lua/autorun/server/sv_cfc_notifications.lua

```diff
@@ -0,0 +1,57 @@
+util.AddNetworkString( "CFC_ReceivePopupNotification" )
+
+CFCNotifications = CFCNotifications or {}
+
+CFCNotifications._timerNameCount = 1
+
+local function getTimerName()
+    local newName = "notification-timer-" .. tostring( CFCNotifications._timerNameCount )
+
+    CFCNotifications._timerNameCount = CFCNotifications._timerNameCount + 1
+
+    return newName
+end
+
+local function makePopup( message, filter )
+    local getPlayers = filter or player.GetAll
+
+    local players = getPlayers()
+
+    net.Start( "CFC_PopupNotification" )
+        net.WriteString( message )
+    net.Send( players )
+end
+
+CFCNotifications._registerNotification = function( notificationType, message, delay, recurring, filter )
```

Comment:
Is the `_` us politely asking people not to mess with it, or does Lua have some form of encapsulation?

Reply:
Just seems like good practice to underscore methods that no one else would intend to use